Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -429,20 +464,24 @@ void JitArm64::WriteExit(Arm64Gen::ARM64Reg dest, bool LK, u32 exit_address_afte
   }
 }
 
-void JitArm64::FakeLKExit(u32 exit_address_after_return)
+void JitArm64::FakeLKExit(u32 exit_address_after_return, ARM64Reg preloaded_after_reg)
```

Comment:
I'm guessing the `preloaded_after_reg` name is based on the `after_reg` name, but it gets really confusing if you just read the function signature without reading the implementation of this function. At first I thought the name was talking about something that got preloaded after a register... Which doesn't really make much sense.

Ideally you should make it clear that this has a connection to the other parameter. For instance, you could name this parameter `exit_address_after_return_reg`. Though I do suppose that name is on the long side.